.circleci/config.yml

```diff
@@ -398,99 +407,151 @@ workflows:
   version: 2
   build_and_test:
     jobs:
+      - build-local:
```

Comment:
why we are building a docker image for every branch? maybe let add some comments with an explanation

Reply:
Yeah this was an idea I had so that any tests inside of the docker image can be run along with the unit tests.